src/server/status/metrics_collector/metrics.js

```diff
@@ -40,14 +69,21 @@ export class Metrics {
             '5m': get(hapiEvent, 'osload.1'),
             '15m': get(hapiEvent, 'osload.2')
           }
+        },
+        mem: {
+          free_in_bytes: os.freemem(),
+          total_in_bytes: os.totalmem()
         }
       },
       response_times: {
-        avg_in_millis: get(hapiEvent, ['responseTimes', port, 'avg']),
-        max_in_millis: get(hapiEvent, ['responseTimes', port, 'max'])
+        // TODO: rename to use `_ms` suffix per beats naming conventions
+        avg_in_millis: isNaN(avgInMillis) ? undefined : avgInMillis, // convert NaN to undefined
```

Comment:
thoughts on getting this over with now?  if it's a new api we don't have to worry about breaking changes yet

Reply:
This module is still used by the status API though, so a change in the field name here would break that.